README.md

```diff
@@ -85,7 +85,7 @@ Clone this repository to your desired folder:
 👤 **Emeka Emmanuel**
 
 - GitHub: [@emekaemmanuel](https://github.com/emekaemmanuel)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/emeka-ugboaja-167820226)
+- LinkedIn: [Emeka Ugboaja](https://linkedin.com/in/emeka-ugboaja-167820226)
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p> 
```

Comment:
- [ ] Please you should add some text on top of these two images to match the design 
your design 
![image](https://user-images.githubusercontent.com/74744787/210815414-61da18ac-93a8-4314-a1d8-0e5fc48cec64.png)
Behance 
![image](https://user-images.githubusercontent.com/74744787/210815521-1c0664bd-867d-4a68-a853-fdf5c60dc5ea.png)

Reply:
✔️